### PR TITLE
About contact

### PR DESCRIPTION
**Pull request by dantoth24**
Polished the about and contact pages. Added a new model to handle the adding and deleting of committee members from the about page.

### CODE REVIEW

File: app/views/about/index.html.erb [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
-  <br>
-  <br>
-  <br>
-  <h2 class="container-fluid">Our Mission </h2>
-  <br>
-  <br>
-  <h2>
-    To serve as a social network of like-minded graduate students to promote women in chemistry.
-  </h2>
+<div class="jumbotron">
```
Comment (nicolepb) [inline app/views/about/index.html.erb:1]: Nice polishing and view, looks clean and professional

File: app/views/messages/new.html.erb [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-      <p>
-        Please send us feedback, comments, or any questions below.
-      </p>
-  </div>
-    <%= form_for @message, url: create_message_path do |f| %>
-
-      <%= @message.errors.full_messages.join(', ') %>
+<head>
+  <title>Contact</title>
+  <link href="form-validation.css" rel="stylesheet">
```
Comment (nicolepb) [inline app/views/messages/new.html.erb]: The emails are not going through, you get this error:
ActionController::RoutingError (No route matches [GET] "/form-validation.css")
Make sure to see that the email message is outputted in the terminal when you press send.

File: app/models/leader.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class Leader < ApplicationRecord
```
Comment (nicolepb) [inline app/models/leader.rb:1]: model looks good, potentially add documentation, or make sure it's easy to explain to them how to use it.

**Review by nicolepb — CHANGES_REQUESTED** [review]
Forms look nice but the messages don't send. You are getting an error. check comments for error

File: app/views/messages/new.html.erb [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+      <%= bootstrap_form_for @message, url: create_message_path do |f| %>
 
-      <%= f.text_field :name, placeholder: 'Name', class: "form-control", style: "margin-top: 10px;" %>
+        <%= f.text_field :name, class: "form-control"%>
 
-      <%= f.email_field :email, placeholder: 'Email',class: "form-control", style: "margin-top: 10px;" %>
+        <%= f.email_field :email, placeholder: "you@example.com", class: "form-control" %>
 
-      <%= f.text_area :body, placeholder: 'Insert message here.', class: "form-control", style: "margin-top: 10px;" %>
+        <%= f.text_area :body, class: "form-control" %>
```
Comment (harperiscoding) [inline app/views/messages/new.html.erb]: this should be larger: more like a square

File: app/views/messages/new.html.erb [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
 
-      <%= f.submit 'Send', class: "btn btn-primary form-control", style: "margin-top: 10px;" %>
-    <% end %>
-    <p>
```
Comment (harperiscoding) [inline app/views/messages/new.html.erb:21]: make sure these static pages always take up the max height of the page, so the footer renders below them.

**Review by harperiscoding — CHANGES_REQUESTED** [review]

File: app/views/messages/new.html.erb (hunk 5 above)
Comment (dantoth24) [reply app/views/messages/new.html.erb:21]: I believe the default height is already 100%, but I'll add more styling to make sure.

**Review by harperiscoding — CHANGES_REQUESTED** [review]
Looks 99% good; I'm gonna add some color variables and push that to master, so if you can use those and get the colors into all those pages I think it's good to go. well done

**Review by harperiscoding — APPROVED** [review]

**Pull request merged**